audit/README.md

```diff
@@ -0,0 +1,100 @@
+# cool-accounts - audit subdirectory #
+
+This subdirectory contains Terraform code to provision the COOL
+"audit" account.  It creates:
+
+* An IAM role that allows sufficient permissions to provision all AWS
+  resources in this account.  This role has a trust relationship with
+  the users account.
+
+## Bootstrapping this account ##
+
+Note that this account must be bootstrapped.  This is because there is
+no IAM role that can be assumed to build out these resources.
+Therefore you must first apply this Terraform code with programmatic
+credentials for AWSAdministratorAccess as obtained for the COOL Audit
+account from the AWS SSO page.
+
+To do this, follow these steps:
+
+1. Comment out the `assume_role` block for the "default" provider in
+   `providers.tf` and directly below that uncomment the line `profile
+   = "cool-audit-account-admin"`.
+1. Create a new AWS profile called `cool-audit-account-admin`
+   in your Boto3 configuration using the "AWSAdministratorAccess"
+   credentials (access key ID, secret access key, and session token)
+   as obtained from the COOL audit account:
+
+   ```console
+   [cool-audit-account-admin]
+   aws_access_key_id = <MY_ACCESS_KEY_ID>
+   aws_secret_access_key = <MY_SECRET_ACCESS_KEY>
+   aws_session_token = <MY_SESSION_TOKEN>
+   ```
+
+1. Create a Terraform workspace (if you haven't already done so) by running
+   `terraform workspace new <workspace_name>`
+1. Create a `<workspace_name>.tfvars` file with all of the required
+   variables (see [Inputs](#Inputs) below for details):
+
+   ```console
+   this_account_id = "111111111111"
+   user_account_id = "222222222222"
```

Comment:
This should be `users_account_id` to match the variable name

Reply:
Fixed in 45af422.